### PR TITLE
I propose to add your own widget Menu

### PR DESCRIPTION
**Pull request by mazurva**
This solution allows to simplify the use of extensions. And also will get rid of duplicate items in the html code.

### CODE REVIEW

**Comment by githubjeka** [comment]
`.gitignore` not needed. 
All the rest is syntax impurity.
I'm neutral in this.

@schmunk42 decide.

**Comment by githubjeka** [comment]
https://github.com/dmstr/yii2-adminlte-asset/blob/master/example-views/phundament/app/layouts/_sidebar.php#L87

hmm.. phundament not used widget Menu. Ok, to schmunk42 no questions.

@Faryshta and others what do you think?

File: example-views/yiisoft/yii2-app/layouts/left.php [hunk 1]
```diff
@@ -30,7 +26,7 @@
         </form>
         <!-- /.search form -->
 
-        <?= Menu::widget([
+        <?= dmstr\widgets\Menu::widget([
             'encodeLabels' => false,
             'options' => ['class' => 'sidebar-menu'],
             'submenuTemplate' => '<ul class="treeview-menu">{items}</ul>',
```
Comment (githubjeka) [inline example-views/yiisoft/yii2-app/layouts/left.php]: can be delete.

File: example-views/yiisoft/yii2-app/layouts/left.php [hunk 2]
```diff
@@ -30,7 +26,7 @@
         </form>
         <!-- /.search form -->
 
-        <?= Menu::widget([
+        <?= dmstr\widgets\Menu::widget([
             'encodeLabels' => false,
```
Comment (githubjeka) [inline example-views/yiisoft/yii2-app/layouts/left.php]: encodeLabels & options can be transfer to init `dmstr\widgets\Menu`

**Comment by githubjeka** [comment]
Thanks

**Comment by githubjeka** [comment]
@schmunk42 please add new tag 2.1.4. Thank

**Comment by Faryshta** [comment]
it loks amazing to me

**Pull request merged**